lib/view/john_gerrard_live_performance.dart

```diff
@@ -77,20 +52,62 @@ class _JohnGerrardLivePerformanceWidgetState
       create: (_) => RetryCubit(),
       child: BlocBuilder<RetryCubit, int>(
         builder: (context, attempt) {
-          if (attempt > 0) {
-            _renderingWidget?.dispose();
-            _renderingWidget = null;
+          final medium = widget.exhibition.series!.first.medium;
+          Widget renderingWidget;
+          switch (medium) {
+            case RenderingType.image:
+              renderingWidget = InteractiveViewer(
+                minScale: 1,
```

Comment:
can you refactor this repeated code
renderingWidget = InteractiveViewer(
                minScale: 1,
                maxScale: 4,
                child: Center(

Reply:
I don't think we need to do that. We create a specific widget for each medium.